core/src/main/java/io/questdb/cutlass/pgwire/PGConnectionContext.java

```diff
@@ -1844,7 +1844,6 @@ private void processCompiledQuery(SqlCompiler compiler, CompiledQuery cq)
             case CompiledQuery.COPY_LOCAL:
                 // uncached
                 queryTag = TAG_COPY;
-                sendCopyInResponse(compiler.getEngine(), cq.getTextLoader());
```

Comment:
sending this confuses clients. they do not expect the copy subprotocol response.